### DISCUSSION TITLE
git提交不上去

### DISCUSSION THREAD

**Discussion by wlisdog** [wechaty/wechaty#2109] [General]
> wechaty.js.org@0.6.84 lint:yml /Users/yangdagang/Desktop/GHS/wechaty.js.org
> true || yamllint jekyll/_data/*.yml

npm ERR! Git working directory not clean.
npm ERR! D package-lock.json

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/yangdagang/.npm/_logs/2020-12-15T10_59_32_200Z-debug.log
Failed to exec pre-push hook script
error: failed to push some refs to 'git@github.com:wlisdog/wechaty.js.org.git'

**Comment by huan** [comment]
try to use `NO_HOOK=1 git push`
